src/ng/compile.js

```diff
@@ -3052,8 +3052,8 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
 
       $compileNode.empty();
 
-      $templateRequest(templateUrl)
-        .then(function(content) {
+      $templateRequest(templateUrl).
```

Comment:
I did a search of the `src` folder for `/\w+\.$/` and didn't find any - although there were lots of matches for sentences in the docs and I might have missed the code ones.